ConwaysGameOfLife.Tests/CellShould.cs

```diff
@@ -42,16 +42,44 @@ public void WhenInitialized_SetLocation(int x, int y)
         }
 
         [Theory]
-        [InlineData(CellState.Dead, 0, 1, CellState.Dead, 0, 1, true)]
-        [InlineData(CellState.Dead, 0, 1, CellState.Alive, 0, 1, false)]
-        public void GivenTwoCells_CellStateAndLocationAreEqual(CellState state1, int x1, int y1,  CellState state2, int x2, int y2, bool expected) 
+        [InlineData(CellState.Dead, 0, 1, CellState.Dead, 0, 1)]
+        [InlineData(CellState.Alive, 0, 1, CellState.Alive, 0, 1)]
+        public void GivenTwoCells_CellStateAndLocationAreEqual(CellState state1, int x1, int y1,  CellState state2, int x2, int y2) 

```

Comment:
good, can we go one step further given that we're now passing the exact same thing to the constructors of each of the two cells?

also a naming suggestion
```suggestion
        [InlineData(CellState.Dead, 0, 1)]
        [InlineData(CellState.Alive, 0, 1)]
        public void BeEqual_GivenSameCellStateAndLocation(CellState state, int x, int y) 
```